### ISSUE TITLE
Final Checklist

### ISSUE THREAD

**Issue by zhijiong** [fangshuye98/CA_WildFire_ML#12]
Final Checklist:

- [x] 1. Basic reference maps: 

- Elevation
- Vegetation
- Subregion
- Population, Road, Powerline

- [x] 2. Summary tables: 

- Vegetation type proportions
- Breakdown of human vs non-human ignitions
- Categories of human-related ignition causes
- Variables used in modeling

- [x] 3. Schematic diagram for the model design

- [ ] 4. Model results:

(1) Map: Predicted 99.9th percentile ignition probability vs. observed fire frequency
(2) ROC curve
(3) Bar chart: Relative feature importance
(4) Heatmap: Temporal variation in relative feature importance across training sets
(5) Time series: Monthly predicted vs. observed ignition counts
	•	Threshold: % of pixels ignited
	•	Subregion-wise exploration
	•	Special focus on July overprediction: Map of July overprediction hot spots

- [ ] 5. Climatology test (CLIM vs CTL):

- Use multi-year daily mean weather data (2001–2020) as test input in the CLIM experiment
(1) Time series: Predicted 99.9th percentile ignition probability (CLIM vs. CTL), with annual PDSI as background
(2) Difference maps:
	•	Annual
	•	Wet Season (Oct–Apr)
	•	Early Fire Season (May–Jun)
	•	Peak Fire Season (Jul–Sep)
(3) Boxplots: Subregional distribution of prediction differences in different seasons
•	Investigate linkage to PDSI patterns

To-do Items

- [x] 1. Model adjustment: Exclude pixels classified as Desert, Wetland, and Barren

- [ ] 2. Add SHAP (no seasonal split):
 	•	Full-region
	•	Subregion-specific

- [ ] 3. Climatology test: Use the original LAI in the CLIM scenario

**Comment by fangshuye98** [comment]
model_version = "Extended_Data_Water_Year_no_riparian_desert_wetland_barren_group_veg"


| Result | Data Path                                                                 | File Name                          | Note                                |
|------------|---------------------------------------------------------------------------|------------------------------------|-------------------------------------|
| Original Prediction| Clean_Data/Model_Data/Evaluation/Features_w_Label_w_pred/{model_version}/parquet | {yr}_predictions.parquet          |          |
| Climatology test | Clean_Data/Model_Data/Evaluation/Features_w_Label_w_pred/{model_version}/CLIM               | {yr}_predictions.parquet       | new prediction col: predictions_update             |
| ROC Plot | Plot/{model_version}/ROC |     | Available in Github |
| Feature Importance Plot | /Plot/{model_version}/FeatureImportance/TOP10FromEachYear                                | top_10_features_importance_pct_over_years.png          |  Available in Github   |
| Shap Plot | Plot/{model_version}/Shap/AllFireAllSeason/Mean_Abs_SHAP_Bar                               | SHAP_Bar_{yr}_{area}_greater_0.png          |  Area contains 'All' and other 5 subregions; Available in Github   |
| Data for plotting Shap | Summary_Data/Shap_Value/{model_version}/ | shap_values_top_15.csv | Available in Github |
| Fire Num Summary for Shap | Summary_Data/Shap_Value/{model_version}/ | fire_num_by_year_area_season.csv | Available in Github|
| Data for plotting feature importance | Summary_Data/Feature_Importance/{model_version}/ | feature_importance_all_years.csv | Available in Github|

Pls refer to https://github.com/fangshuye98/CA_WildFire_ML/issues/10#issuecomment-2986612492 for summary data description (same as before)

**Comment by fangshuye98** [comment]
added top 25

https://github.com/fangshuye98/CA_WildFire_ML/blob/main/Summary_Data/Shap_Value/Extended_Data_Water_Year_no_riparian_desert_wetland_barren_group_veg/shap_values_top_25.csv

**Comment by fangshuye98** [comment]
Model parameters for year 2001:
{
    "generic_param": {
        "device": "cpu",
        "fail_on_invalid_gpu_id": "0",
        "n_jobs": "0",
        "nthread": "0",
        "random_state": "0",
        "seed": "0",
        "seed_per_iteration": "0",
        "validate_parameters": "1"
    },
    "gradient_booster": {
        "gbtree_model_param": {
            "num_parallel_tree": "1",
            "num_trees": "100"
        },
        "gbtree_train_param": {
            "process_type": "default",
            "tree_method": "hist",
            "updater": "grow_quantile_histmaker",
            "updater_seq": "grow_quantile_histmaker"
        },
        "name": "gbtree",
        "specified_updater": false,
        "tree_train_param": {
            "alpha": "0",
            "cache_opt": "1",
            "colsample_bylevel": "1",
            "colsample_bynode": "1",
            "colsample_bytree": "1",
            "eta": "0.300000012",
            "gamma": "0",
            "grow_policy": "depthwise",
            "interaction_constraints": "",
            "lambda": "1",
            "learning_rate": "0.300000012",
            "max_bin": "256",
            "max_cat_threshold": "64",
            "max_cat_to_onehot": "4",
            "max_delta_step": "0",
            "max_depth": "6",
            "max_leaves": "0",
            "min_child_weight": "1",
            "min_split_loss": "0",
            "monotone_constraints": "()",
            "refresh_leaf": "1",
            "reg_alpha": "0",
            "reg_lambda": "1",
            "sampling_method": "uniform",
            "sketch_ratio": "2",
            "sparse_threshold": "0.20000000000000001",
            "subsample": "1"
        },
        "updater": [
            {
                "hist_train_param": {
                    "debug_synchronize": "0",
                    "max_cached_hist_node": "65536"
                },
                "name": "grow_quantile_histmaker"
            }
        ]
    },
    "learner_model_param": {
        "base_score": "1.2865476E-1",
        "boost_from_average": "1",
        "num_class": "0",
        "num_feature": "31",
        "num_target": "1"
    },
    "learner_train_param": {
        "booster": "gbtree",
        "disable_default_eval_metric": "0",
        "multi_strategy": "one_output_per_tree",
        "objective": "binary:logistic"
    },
    "metrics": [
        {
            "name": "logloss"
        }
    ],
    "objective": {
        "name": "binary:logistic",
        "reg_loss_param": {
            "scale_pos_weight": "1"
        }
    }
}

@zhijiong
